python-checks/src/test/java/org/sonar/python/checks/FloatingPointEqualityCheckTest.java

```diff
@@ -72,10 +72,10 @@ void quickfixInequality(){
     String compliant =
       "import math\n" +
       "def foo(a,b):\n" +
-      "    if !math.isclose(a - 0.1, b, rel_tol=1e-09, abs_tol=1e-09):\n" +
+      "    if not math.isclose(a - 0.1, b, rel_tol=1e-09, abs_tol=1e-09):\n" +
       "        ...";
     PythonQuickFixVerifier.verify(check, noncompliant, compliant);
-    PythonQuickFixVerifier.verifyQuickFixMessages(check, noncompliant, "Replace with !math.isclose().");
+    PythonQuickFixVerifier.verifyQuickFixMessages(check, noncompliant, "Replace with not math.isclose().");
```

Comment:
Maybe a matter of personal preference, but I have been putting parentheses around the suggested expressions. I think it integrated better into the message:
`"Replace with \"not math.isclose()\"."`